### ISSUE TITLE
Random test failure: Update: Update core functionality (UpdateCoreTestCase)

### ISSUE THREAD

**Issue by Gormartsen** [backdrop/backdrop-issues#1965]
Relate to #1478
Probably connected with: #1254

```
Update: Update core functionality (UpdateCoreTestCase)

fail "Failed to get available update data for one project." found only once update.test:226
```

https://zen.ci/backdrop/backdrop/test/test-php53-backdrop_backdrop_1434-7215

**Comment by klonos** [comment]
Lets close this as a duplicate of #1254

**Comment by klonos** [comment]
...actually lets not 😅 

#1254 is about the fact that checking for updates does fail randomly, whereas this issue here is about the failing test ...which happens because of #1254 ...which makes me question the value of that specific test.

**Comment by quicksketch** [comment]
I think this may have been fixed in #5205 

The fixing of batch processing may also have fixed this failure with update checking, because it is depend on batch processing.

**Comment by klonos** [comment]
Surely hope so 🤞🏼 ...lets leave this open, to allow a few weeks/months of trialing the batch fixes with GHA, and close if we have not seen this surface in the meantime.

**Comment by quicksketch** [comment]
I haven't seen this issue in a very long time, so I believe this particular one is closed.
